### DISCUSSION TITLE
can't run et legacy

### DISCUSSION THREAD

**Discussion by duchu-d2** [etlegacy/etlegacy#2373] [General]
I'm trying to run it on win11:

       0 ET Legacy v2.81.1 win-x86 Mar 12 2023
       0 Zone megs: 64
       0 INFO: fs_game now defaults to 'legacy' mod instead of 'etmain'
       0 ----- Initializing Filesystem --
       0 Current working directory:
       0     C:\Program Files (x86)\ETLegacy
       0 Current search path:
       0     D C:\Users\rafal\Documents\ETLegacy\legacy
       0     D .\legacy
       0     D C:\Users\rafal\Documents\ETLegacy\etmain
       0     D .\etmain
       0     P .\legacy\legacy_v2.81.1.pk3 (1407 files)
       0     P .\legacy\legacy_v2.81.0.pk3 (1403 files)
       0     P .\legacy\legacy_v2.80.2.pk3 (1390 files)
       0     P .\etmain\pak2.pk3 (22 files)
       0     P .\etmain\pak1.pk3 (10 files)
       0     P .\etmain\pak0.pk3 (3725 files)
       0 
       0 7957 files in pk3 files
       0 --------------------------------
       0 Creating PID file 'profiles/duchu/profile.pid'
       0 ----- Database Initialization --
       0 SQLite3 libversion 3.36.0 - database URI 'etl.db' - in file
       0 ... loading existing database 'C:\Users\rafal\Documents\ETLegacy\etl.db'
       0 ... database file 'C:\Users\rafal\Documents\ETLegacy\etl.db' loaded
       0 SQLite3 ETL: database init #2 C:\Users\rafal\Documents\ETLegacy\etl.db in [0] ms - autocommit 1
       0 SQLite3 ETL: DB schema version #2 is up to date
       0 --------------------------------
       0 execing default.cfg
       0 execing profiles/duchu/etconfig.cfg
       0 com_zoneMegs will be changed upon restarting. New value: '1024'
       0 couldn't exec autoexec.cfg
       0 Hunk_Clear: reset the hunk ok
       0 Not logging server attacks to disk
       0 ----- Client Initialization ----
       0 ETKEY found
       0 Available client translations: Afrikaans Bulgarian Catalan Czech Danish German Greek English Esperanto Spanish Finnish French Irish Hebrew Hungarian Italian Korean Dutch Norwegian Polish Portuguese Romanian Russian Slovak Slovenian Albanian Serbian Swedish Turkish
       0 Available mod translations: Afrikaans Bulgarian Catalan Czech Danish German Greek English Esperanto Spanish Finnish French Irish Hebrew Hungarian Italian Korean Dutch Norwegian Polish Portuguese Romanian Russian Slovak Slovenian Albanian Serbian Swedish Turkish
       0 Language set to English
       0 --------------------------------
       0 Winsock initialized
       0 Hostname: LAPTOP-7JJCC1BE
       0 IP: 192.168.0.241
       0 IP6: ::1
       0 IP: 10.14.0.2
       0 Opening IP6 socket: [::]:27960
       0 WARNING: NET_IP6Socket: bind: WSAEADDRINUSE
       0 Opening IP6 socket: [::]:27961
       0 Opening IP socket: 0.0.0.0:27960
       0 WARNING: NET_IPSocket - bind: WSAEADDRINUSE
       0 Opening IP socket: 0.0.0.0:27961
       0 Network initialized
       0 ----- Initializing Renderer ----
       0 SDL build version 2.26.4 - link version 2.26.4
       0 SDL initialized driver "windows"
       0 Initializing OpenGL display
       0 Estimated display aspect: 1.600
       0 ...setting mode -2: 1707x1067
       0 Using 24 color bits, 24 depth, 0 stencil display
       0 Available modes [26]: '1280x800 1440x900 1680x1050 1920x1200 2560x1600 1280x768 1360x768 1280x720 1366x768 1600x900 1920x1080 2048x1152 2560x1440 640x480 800x600 1024x768 1152x864 1280x960 1400x1050 1600x1200 1792x1344 1856x1392 1920x1440 2048x1536 1280x1024 1280x600'
       0 Using GLEW 2.1.0
       0 GL_VENDOR: Intel
       0 GL_RENDERER: Intel(R) Iris(R) Xe Graphics
       0 GL_VERSION: 4.6.0 - Build 30.0.101.3111
       0 GL_SHADING_LANGUAGE_VERSION: 4.60 - Build 30.0.101.3111
       0 Using vanilla renderer
       0 Initializing OpenGL extensions
       0 ...ignoring GL_EXT_texture_compression_s3tc
       0 ...GL_S3_s3tc not found
       0 ...found OpenGL extension - GL_EXT_texture_env_add
       0 ...found OpenGL extension - GL_ARB_multitexture
       0 GL_VENDOR: Intel
       0 GL_RENDERER: Intel(R) Iris(R) Xe Graphics
       0 GL_VERSION: 4.6.0 - Build 30.0.101.3111
       0 GL_SHADING_LANGUAGE_VERSION: 4.60 - Build 30.0.101.3111
       0 GL_MAX_TEXTURE_SIZE: 16384
       0 GL_MAX_ACTIVE_TEXTURES_ARB: 8
       0 PIXELFORMAT: color(24-bits) Z(24-bit) stencil(0-bits)
       0 MODE: -2, SCREEN: 1707 x 1067 fullscreen (ratio 1.5998) Hz:N/A
       0 GAMMA: hardware w/ 0 overbright bits
       0 texturemode: GL_LINEAR_MIPMAP_LINEAR
       0 picmip: 0
       0 texture bits: 32
       0 multitexture: enabled
       0 compiled vertex arrays: enabled
       0 texenv add: enabled
       0 compressed textures: disabled
       0 Setting up FBO
       0 Initializing Shaders
       0 --------------------------------
       0 ----- Initializing Sound (1) ---
       0 SDL_Init( SDL_INIT_AUDIO )... OK
       0 SDL audio driver is "wasapi".
       0 SDL_AudioSpec:
       0   Format:   AUDIO_S16LSB
       0   Freq:     44100
       0   Samples:  1024
       0   Channels: 2
       0   Silence:  0
       0   Size:     4096
       0 Starting SDL audio callback...
       0 SDL audio initialized.
       0 ----- Sound Info -----
       0 sound system is muted
       0     2 channels
       0 32768 samples
       0    16 samplebits
       0     1 submission_chunk
       0 44100 speed
       0 55B05CC8 dma buffer
       0 No background file.
       0 ----------------------
       0 Sound initialization successfully done
s_backend set to SDL2
       0 --------------------------------
       0 ----- Client Shutdown ----------
       0 Closing SDL audio device...
       0 SDL audio device shut down.
       0 RE_Shutdown( 1 )
       0 SDL input devices shut down.
       0 --------------------------------
       0 PID file removed.
       0 SQLite3 database closed.
Sound buffer failed to allocate 0.1 megs
       0 SDL input devices shut down.

**Comment by u1ps** [comment]
com_soundmegs 96 fixed this for me.
Default  value 160 gives me the same error
https://discord.com/channels/260750790203932672/260750790203932672/1127598209234120744

**Reply by duchu-d2** [reply]
now I have this:
<img width="361" alt="image" src="https://github.com/etlegacy/etlegacy/assets/149110367/5dd3728e-0826-4f30-90f8-df3b6ec646ac">

**Reply by rafal1137** [reply]
Can you tell us which server were you trying to join ?

**Reply by u1ps** [reply]
i had to set hunkmegs 512 and zonemegs 128 and then i was fine. I had exactly same problem as you.

I also started using 64bit clients instead of 32bit

https://discord.com/channels/260750790203932672/260750790203932672/1128000912351318066

**Reply by ensiform** [reply]
What the heck needs 128 zone megs? 🤯
